src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -701,8 +701,8 @@ function copyOptions() {
 
     // Number of recipient setup
     formatNumberBox($currRecipient.val(), NEW_QUESTION);
-    var $prevRadioButtons = $('table[class~="questionTable"]').eq(-2).find('input[name="numofrecipientstype"]');
-    var $currRadioButtons = $('table[class~="questionTable"]').last().find('input[name="numofrecipientstype"]');
+    var $prevRadioButtons = $('div[class~="questionTable"]').eq(-2).find('input[name="numofrecipientstype"]');
```

Comment:
Can you investigate a solution that does not require ```.eq(``` in this case because it assumes that the HTML DOM structure is a certain why